Getting_and_Cleaning_Data/Tidying_Data_with_tidyr/students3.csv

```diff
@@ -2,7 +2,7 @@ name,test,class1,class2,class3,class4,class5
 Sally,midterm,A,,B,,
 Sally,final,C,,C,,
 Jeff,midterm,,D,,A,
-Jeff,final,,E,,C,
```

Comment:
I did not realize that E was a standard letter grade in other countries.  Good to know!